platform/lib/utils/imageOptimizer.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require('path');
+
+const Image = require('@11ty/eleventy-img');
+
+const DEFAULT_IMG_FORMAT = 'jpeg';
+
+const baseDir = path.join(__dirname, '../../..');
```

Comment:
I'd suggest using the project helper to build the paths:

```js
const { project } = require('@lib/utils');

const baseDir = project.absolute('/');
```

Or for this specific case there's also:

```js
const baseDir = project.paths.ROOT;
```